x-pack/plugins/endpoint/public/applications/endpoint/view/alerts/index.tsx

```diff
@@ -207,7 +213,47 @@ export const AlertIndex = memo(() => {
     [setVisibleColumns, visibleColumns]
   );
 
-  const selectedAlertData = useAlertListSelector(selectors.selectedAlertDetailsData);
+  useEffect(() => {
+    if (closeSuccess !== undefined) {
+      notifications.toasts.success({
+        title: (
+          <FormattedMessage
+            id="xpack.endpoint.alerts.closedAlert.successTitle"
+            defaultMessage="Successfully Closed Alert"
```

Comment:
Does it make sense to use a notification in the case of success? Maybe we should only notify on error.

Reply:
the thought is to add a link to the closed alerts tab in the success message, so we'll want to notify on both